content/en/docs/Release Management/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Release Management
+linkTitle: "Release Management"
+weight: 120
+---
+
+## Release Management Guide.
+This document describes the Kyverno release process according to timeline and how to maintain release branches.
+
+### The overall scope of the project
+
+Kyverno simplifies Kubernetes configuration management using policies for security and automation. Kyverno makes it easy for Kubernetes administrators to write and manage Kubernetes policies, and for Kubernetes users (ex., developers) to consume policy results and address issues.
+
+### Release Timelines:
+
+Kyverno uses the Semantic Versioning scheme. Kyverno v1.0.0 was released in Dec 2019. This project follows a given version number MAJOR.MINOR.PATCH.
```

Comment:
Kyverno specifically uses Semver 2.0, from where you've copied the majority of the text from the releases. It'd be best to link to https://semver.org/